{{cookiecutter.project_slug}}/pyproject.toml

```diff
@@ -90,7 +90,11 @@ myst-parser = "^0.18.1"
 {%- elif cookiecutter.documentation_engine == 'jupyter-book' %}
 jupyter-book = "^0.12.3"
 myst-parser = "^0.15.0"
-{%+ endif %}
+{% endif -%}
+
+{%- if cookiecutter.code_security == 'bandit' %}
```

Comment:
@xmnlab Why is this highlighted in red?

Reply:
probably it is just a feature of github rendering this kind of context .. feel free to ignore that